contracts/nodes/VVVNodes.sol

```diff
@@ -240,7 +244,8 @@ contract VVVNodes is ERC721, VVVAuthorizationRegistryChecker {
         uint256 totalUnvestedAmount = _tokenData.unvestedAmount;
         uint256 amountToVestPerSecond = _tokenData.amountToVestPerSecond;
 
-        uint256 timeBasedVestingAmount = (block.timestamp - vestingSince) * amountToVestPerSecond;
+        //inclusive of timestamp which set vestingSince
+        uint256 timeBasedVestingAmount = (block.timestamp - vestingSince + 1) * amountToVestPerSecond;
```

Comment:
It wound up being a fault of the test, so I removed this extra `+1` from the contract and advanced the timestamp one more second in the test.

The trick was that I was advancing "vestingDuration/2 - 1" relative to `vestingSince` instead of "vestingDuration/2", so the change in unvested tokens was not equal to the change in claimed tokens (off by 1e18). 

After fixing this, now the test is performed at exactly halfway through the vesting, so the changes in each quantity are equal.